### DISCUSSION TITLE
How to add a new plugin at https://github.com/pelican-plugins

### DISCUSSION THREAD

**Discussion by anjos** [getpelican/pelican#3415] [Q&A]
Hello,

I'm the happy user of Pelican since a few years.  One of plugins I currently use seem to be outdated [pelican-fontawesome](https://github.com/kura/pelican-fontawesome).  I thought about updating the repo, but it looks abandoned (10 years w/o commits or answers).

I wonder what it would take to move that plugin, after a refresh, to https://github.com/pelican-plugins.  Is there a process?

**Comment by justinmayer** [comment]
Hi André. If you would be willing to help modernize this plugin by way of the provided [Pelican plugin template](https://github.com/getpelican/cookiecutter-pelican-plugin), I would be happy to create a new individual repository under the Pelican Plugins organization and add you there as a collaborator.

I believe updating the plugin structure should be fairly straightforward. Would you please confirm your willingness to help out with that endeavor?

**Reply by anjos** [reply]
I confirm it!  Please let me know what would be the next steps.

**Reply by justinmayer** [reply]
Great! I created a new repository and invited you to join the project. I will soon create an issue at the new repository with some suggested next steps, after which we can take this discussion over there 😊

**Reply by justinmayer** [reply]
Hi André. I created a `migrate` branch at the new repository and opened a [pull request](https://github.com/pelican-plugins/icons/pull/1) in which we can collaborate on this new endeavor. After you have accepted the invitation I sent you, please follow the [Updating the Plugin](https://getpelican.com/blog/namespace-plugin-migration/#updating-the-plugin) section of the _Migrating Plugins to New Organization_ guide, and post comments to this PR with any questions that arise: https://github.com/pelican-plugins/icons/pull/1

Closing this discussion in favor of that new PR. Thanks again for your assistance. Looking forward to collaborating with you! 💫
